xmtp_mls/src/identity.rs

```diff
@@ -132,7 +132,7 @@ impl Identity {
         }
 
         // Register the installation with the server
-        let kp = self.new_key_package(&provider)?;
+        let kp = self.new_key_package(provider)?;
```

Comment:
removing this reference is not related to `rust doc` specifically; other than it no longer generates a lint warning.